dev-lisp/newlisp/newlisp-10.7.5.recipe

```diff
@@ -0,0 +1,57 @@
+SUMMARY="Put the fun back in Lisp"
+DESCRIPTION="newLISP is a LISP-like scripting language for doing things"
+HOMEPAGE="https://www.newlisp.org"
```

Comment:
This URL doesn't work.  Their site isn't configured for SSL.

Reply:
Fixed in https://github.com/haikuports/haikuports/pull/6892